src/store.js

```diff
@@ -40,7 +46,9 @@ export default {
       }
     },
     selectAll (all) {
-      this.selectedRows = all
```

Comment:
You are right, probably the way to go is to just do:
`this.selectedRows = [...all]`
This way we can keep using `===` for comparison